src/main/kotlin/io/kuzzle/sdk/coreClasses/responses/Response.kt

```diff
@@ -80,14 +80,19 @@ class Response : Serializable {
      */
     var type: String? = null
 
+    /**
+     * Response headers
+     */
+    var headers: Map<String?, Any?>? = null
```

Comment:
why not KuzzleMap here ?

Reply:
KuzzleMap is mostly used internally, because it's easier to use the Map<String?, Any?> since it's native than KuzzleMap where people might need to see what's inside.

Reply:
By the way, it's just a info but Kotlin have type alias : https://kotlinlang.org/docs/type-aliases.html
Maybe, in the future, it can be useful to use some of them to avoid to much complex types.